src/communication/azext_communication/manual/_help.py

```diff
@@ -111,3 +111,58 @@
         text: |-
                az communication wait --name "MyCommunicationResource" --resource-group "MyResourceGroup" --deleted
 """
+
+helps['communication identity'] = """
+    type: group
+    short-summary: Commands to manage User Identity for a CommunicationService resource.
+"""
+
+helps['communication identity issue-access-token'] = """
+    type: command
+    short-summary: "Issues a new access token with the specified scopes for a given User Identity. If no User Identity is specified, creates a new User Identity as well."
+    examples:
+      - name: issue-access-token
+        text: |-
+               az communication identity issue-access-token --scope chat
+      - name: issue-access-token with multiple scopes and userid
+        text: |-
+               az communication identity issue-access-token --scope chat voip --userid "8:acs:xxxxxx"
+"""
```

Comment:
What's this command used for? Where will user use the access token?

Reply:
@kairu-ms This command is used for generate an access_token can be used to make call to communication service endpoints. This command help achieve this in azure cli - [Quickstart: Create and manage access tokens](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-csharp)

Reply:
@kairu-ms can you please review?

Reply:
@moirf, If you want to release this extension, you should update the version of the extension and add release history

Reply:
@kairu-ms version and release history updated.